game-7daystodie/index.tsx

```diff
@@ -133,7 +133,7 @@ async function prepareForModding(context: types.IExtensionContext,
     }
     await fs.ensureDirWritableAsync(path.join(directory, 'Mods'));
     const launcher = DEFAULT_LAUNCHER_SETTINGS;
-    launcher.DefaultRunConfig.AdditionalParameters = `-UserDataFolder=${directory}`;
+    launcher.DefaultRunConfig.AdditionalParameters = `-UserDataFolder=${directory}"`;
```

Comment:
Thanks @Senjay-id - although please note that you forgot a double quotation mark - the value should've been set to:
`-UserDataFolder="${directory}"`

Reply:
@IDCs After testing the configuration further, it throws launcher error if we use double quotation mark. I think the only way to fix this is to change how 7d2d parses the configuration in the game's assembly code

`
2024-09-19T17:43:44 0.178 INF Command line arguments: C:\Hub\Steam\steamapps\common\7 Days To Die\7DaysToDie.exe -force-d3d11 -disablenativeinput -nogs -noeac -UserDataFolder=C:\Users\firstname lastname\AppData\Roaming\7daystodie

2024-09-19T17:43:44 0.183 INF UserDataFolder: C:\Users\firstname
`

Reply:
I'll take a look at the game assembly code and see if it's possible to fix the parser

Reply:
@IDCs Alright the change should be able to parse udf path that has whitespace, reminder again that I haven't tested the extension but the expected output should be
`
"AdditionalParameters": "-UserDataFolder=\"C:\\Users\\firstname lastname\\AppData\\Roaming\\7daystodie\""
`